packages/core/src/com/initializers/iframe.ts

```diff
@@ -99,8 +105,10 @@ async function startIframe({ com, iframe, instanceId, src, envReadyPromise }: St
         url.searchParams.set(INSTANCE_ID_PARAM_NAME, instanceId);
         iframe.src = url.href;
 
-        await Promise.race([waitForCancel, waitForLoad(iframe), envReadyPromise]);
-
+        await Promise.race([waitForCancel, envReadyPromise]);
```

Comment:
check what happens when waitForCancel is called? the registration still happens?

Reply:
not something good. I don't want to change it here. I will change when the refactor of the iframe in codux will be fully resolved